### PR TITLE
`tools/importer-rest-api-specs`: migrating SDK Operation Options over to the new Data API SDK Types

### PR DESCRIPTION
**Pull request by tombuildsstuff**
This PR refactors the SDK Operation Options over to the new Data API SDK types - meaning these are now consolidated into a single type.

For the sake of convenience for now I've opted to update the old `resourcemanager` package to reference the new SDK types as the migration goes on - since this allows refactoring components piecemeal rather than needing to migrate off the entire package at a time.

### CODE REVIEW

File: tools/importer-rest-api-specs/components/parser/operations.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ package parser
 
 import (
 	"fmt"
+	"github.com/hashicorp/pandora/tools/data-api-sdk/v1/models"
```
Comment (catriona-m) [inline tools/importer-rest-api-specs/components/parser/operations.go:8]: imports need sorting here

**Review by katbyte — APPROVED** [review]
LGTM 🧨

**Review by catriona-m — APPROVED** [review]
LGTM!

File: tools/importer-rest-api-specs/components/parser/operations.go (hunk 1 above)
Comment (tombuildsstuff) [reply tools/importer-rest-api-specs/components/parser/operations.go:8]: This is fixed in one of the later PRs fwiw

**Pull request merged**